### ISSUE TITLE
FacebookJsonMappingException when fetching page object hours

### ISSUE THREAD

**Issue by AvielNiego** [restfb/restfb#965]
I called 
`facebookClient.fetchObject("10150171327364648", Page.class, Parameter.with("fields", "hours"))`

and got 
`Method threw 'com.restfb.exception.FacebookJsonMappingException' exception.`
`JSON is an array but is being mapped as an object - you should map it as a List instead. Offending JSON is '[{"key":"mon_1_open","value":"11:00"},{"key":"mon_1_close","value":"02:00"},{"key":"mon_2_open","value":"11:00"},{"key":"mon_2_close","value":"23:45"},{"key":"tue_1_open","value":"11:00"},{"key":"tue_1_close","value":"02:00"},{"key":"tue_2_open","value":"11:00"},{"key":"tue_2_close","value":"23:45"},{"key":"wed_1_open","value":"11:00"},{"key":"wed_1_close","value":"02:00"},{"key":"thu_1_open","value":"11:00"},{"key":"thu_1_close","value":"02:00"},{"key":"fri_1_open","value":"11:00"},{"key":"fri_1_close","value":"02:00"},{"key":"sat_1_open","value":"11:00"},{"key":"sat_1_close","value":"02:00"},{"key":"sun_1_open","value":"11:00"},{"key":"sun_1_close","value":"02:00"}]'.`

using facebook v3.2 and restfb version 2.16.0

**Comment by nbartels** [comment]
@AvielNiego great you find this one, Facebook changed the structure of the field sometime. We changed the structure completely to make the access to the data more easy.

**Comment by mknoebel** [comment]
Hello,
I have got a similar Problem while trying to fetch the hours of my childPages.

My call:
`facebookClient.fetchConnection("me/locations", Page.class, Parameter.with("fields", "store_number, store_location_descriptor, phone, website, location, hours"));`

My Exception:
`Exception in thread "main" com.restfb.exception.FacebookJsonMappingException: JSON is an object but is being mapped as a list instead. Offending JSON is '{"mon_1_open":"08:00","mon_1_close":"16:00","tue_1_open":"08:00","tue_1_close":"16:00","wed_1_open":"08:00","wed_1_close":"16:00","thu_1_open":"08:00","thu_1_close":"16:00","fri_1_open":"08:00","fri_1_close":"16:00"}'.`

I am using Facebook V3.2 and RestFB Version 2.18.0
Btw. I just started working with your RestFB library so I apologize if the answer to my Issue is obvious.
